Changelog.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+### 1.1.3
+
+**Image:**
+- k8s.gcr.io/ingress-nginx/controller:v1.1.3@sha256:31f47c1e202b39fadecf822a9b76370bd4baed199a005b3e7d4d1455f4fd3fe2
+
+This release bumps Alpine to 3.14.4 and nginx to 1.19.10 
+
+_Changes:_
+
+[8403](https://github.com/kubernetes/ingress-nginx/pull/8403) Add execute permissions to nginx image entrypoint.sh
+[8392](https://github.com/kubernetes/ingress-nginx/pull/8392) fix document for monitoring
+[8386](https://github.com/kubernetes/ingress-nginx/pull/8386) downgrade to 3.14.4 and fix tag
+[8379](https://github.com/kubernetes/ingress-nginx/pull/8379) bump luarocks to 3.8.0
+[8368](https://github.com/kubernetes/ingress-nginx/pull/8368) Updated semver in install docs URLs
+[8360](https://github.com/kubernetes/ingress-nginx/pull/8360) Bump github.com/stretchr/testify from 1.7.0 to 1.7.1
+[8343](https://github.com/kubernetes/ingress-nginx/pull/8343) Bump alpine base to 3.15
```

Comment:
We need to remove this line as it is misleading that we bumped to alpine 3.15. We did not